cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,286 @@
+            password="<PASSWORD>",
+        )
+        self.client.force_authenticate(user=self.user)
+
+        self.actor_John = Actor.objects.create(
+            first_name="John",
+            last_name="Doe",
+        )
+        self.actor_Illia = Actor.objects.create(
+            first_name="illia",
+            last_name="Doe",
+        )
+
+        self.genre_horror = Genre.objects.create(name="horror")
+        self.genre_drama = Genre.objects.create(name="drama")
+
+        self.cinema_hall = CinemaHall.objects.create(
+            name="Cinema Hall",
+            rows=12,
+            seats_in_row=4,
+        )
+        self.movie_vinston = Movie.objects.create(
+            title="vinston",
+            description="Test movie",
+            duration=6,
+        )
+        self.movie_apalon = Movie.objects.create(
+            title="apalon",
+            description="Test movie",
+            duration=6,
+        )
+        self.movie_japon = Movie.objects.create(
+            title="japon",
+            description="Test movie",
+            duration=6,
+        )
+
+        self.movie_apalon.actors.add(self.actor_John)
+        self.movie_apalon.genres.add(self.genre_horror)
+
+        self.movie_vinston.actors.add(self.actor_Illia)
+        self.movie_vinston.genres.add(self.genre_drama)
+
+        self.movie_japon.actors.add(self.actor_John)
+        self.movie_japon.genres.add(self.genre_drama)
+
+        self.movie_session_2024 = MovieSession.objects.create(
+            show_time=datetime.datetime(2024, 1, 1, 12, 0, 0),
+            movie=self.movie_apalon,
+            cinema_hall=self.cinema_hall,
+        )
+        self.movie_session_2023 = MovieSession.objects.create(
+            show_time=datetime.datetime(2023, 1, 1, 12, 0, 0),
+            movie=self.movie_vinston,
+            cinema_hall=self.cinema_hall,
+        )
+
+    def test_movie_if_simple_user(self):
+        movie_response = self.client.get(MOVIE_LIST_URL)
+
+        movies = Movie.objects.all()
+
+        movie_serializer = MovieListSerializer(movies, many=True)
+
+        self.assertEqual(movie_response.status_code, status.HTTP_200_OK)
+
+        self.assertEqual(movie_serializer.data, movie_response.data)
+
+    def test_movie_session_if_simple_user(self):
+        movie_sessions_response = self.client.get(MOVIE_SESSION_LIST_URL)
+
+        movie_sessions = (
+            MovieSession.objects.all()
+            .select_related("movie", "cinema_hall")
+            .annotate(
+                tickets_available=(
+                    F("cinema_hall__rows") * F("cinema_hall__seats_in_row")
+                    - Count("tickets")
+                )
+            )
+        )
+
+        movie_session_serializer = MovieSessionListSerializer(movie_sessions, many=True)
+
+        self.assertEqual(movie_sessions_response.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_session_serializer.data, movie_sessions_response.data)
```

Comment:
This test could be improved by adding assertions for the specific fields in the response. This would make your test more robust and likely to catch regressions.